src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -218,6 +229,7 @@ const getAvailablePaymentMethods = (
   let paymentMethod: CommercePaymentMethodEnum = "CREDIT_CARD"
   const paymentMethods = [
     <BorderedRadio
+      id="payment-method"
```

Comment:
**suggestion:** Depending on the available payment methods, this component can render up to 3 DOM elements with the same `id` attribute, which are expected to be unique. Please consider using the `class` attribute instead, or renaming the IDs to be more specific (ex. `payment-method-credit-card`).

Since these IDs are only being used for tests, my preference would be to use a more specific `data-test-id` attribute for each